src/Benchmarks/Data/Fortune.cs

```diff
@@ -19,10 +19,10 @@ public class Fortune : IComparable<Fortune>, IComparable
         public int _Id { get; set; }
 
         [Column("message")]
-        [StringLength(2048)]
+        [MaxLength(2048)]
         [IgnoreDataMember]
         [Required]
-        public string Message { get; set; }
+        public byte[] Message { get; set; }
```

Comment:
Not sure that's valid in terms of requirements.

Reply:
It is. Nothing wrong with UTF8 encoded bytes for the message property.